src/components/task-plan/homework/exercise-controls.cjsx

```diff
@@ -68,24 +68,28 @@ ExerciseControls = React.createClass
     else if @props.canAdd
       <BS.Button bsStyle="default"
         className="-add-exercises"
-        onClick={@props.addClicked}>Add More...</BS.Button>
+        onClick={@props.addClicked}>+ Add More Sections</BS.Button>
     else
```

Comment:
We can't use the bare word `Sections` or `Periods`.  You can use the `CourseGroupingLabel` to switch between the two depending on the course type.